test/metabase/query_analysis/native_query_analyzer_test.clj

```diff
@@ -85,17 +90,20 @@
   (testing "simple query matches"
     (let [sql "select id from venues"]
       (is (= [(field-reference :venues :id)] (field-refs sql)))
-      (is (= [(table-reference :venues)] (table-refs sql))))))
+      (is (= [(table-reference :venues)] (table-refs sql)))
+      (is (= [(table-reference :venues)] (basic-table-refs sql))))))
```

Comment:
Copy paste won't scale, we'll want so generic way to run these tests over all the analysis modes.

Reply:
I'm going to wait until we support `:basic-select` mode, then loop over the modes.